### DISCUSSION TITLE
How can I work for free and unlimited after the colab limit is exceeded?

### DISCUSSION THREAD

**Discussion by kisisel409** [community/community#148062] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I didn't know there were limitations even though I was a colab pro member. This one is sold out. What should I do? I don't want to sign up again. Is there a fooocus that I can work with unlimitedly?

**Comment by mecodeatlas** [comment]
Hi @kisisel409, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Comment by AyoubMabrouk** [comment]
I suggest you start using Jupyter Notebooks or Kaggle Kernels instead, personally I found them more useful then colab.

**Reply by kisisel409** [reply]
Thank you, i am very new for this platform and i need some help.

**Reply by jatinprajapat40-crypto** [reply]
me too i am also very new for theese platforms like jupyter notebooks or kaggle kernels

**Comment by mecodeatlas** [comment]
Hey, @kisisel409! You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!
